cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/prometheus/PrometheusMetricSamplerTest.java

```diff
@@ -80,22 +80,6 @@ public void setUp() {
         _prometheusQueryMap = prometheusQuerySupplier.get();
     }
 
-    @Test(expected = ConfigException.class)
-    public void testConfigureWithPrometheusEndpointNoPortFails() throws Exception {
-        Map<String, Object> config = new HashMap<>();
-        config.put(PROMETHEUS_SERVER_ENDPOINT_CONFIG, "http://kafka-cluster-1.org");
-        addCapacityConfig(config);
-        _prometheusMetricSampler.configure(config);
-    }
-
-    @Test(expected = ConfigException.class)
-    public void testConfigureWithPrometheusEndpointNegativePortFails() throws Exception {
```

Comment:
the names of these two tests don't sound like they're related to what this change is doing, what's the reason to remove them?

Reply:
https://github.com/grafana/cruise-control/commit/ae04ed7a54bb3cda17350c307ebd648e96ed786a actually broke these tests (negative/no explicit port should be valid) but I hadn't been running repo-wide tests till now so I never noticed; this was just to get all tests passing.